AFQ/_fixes.py

```diff
@@ -67,7 +72,12 @@ def _generate_streamlines(self):
                 else:
                     parts = (B[stepsB - 1:0:-1], F[:stepsF])
                     streamline = np.concatenate(parts, axis=0)
-                yield streamline
+
+                len_sl = len(streamline)
+                if len_sl < self.min_length or len_sl > self.max_length:
```

Comment:
Can we make this take into account the step size, so that the length is in mm, rather in the number of steps?